webui/react/src/components/NavigationSideBar.tsx

```diff
@@ -152,8 +154,9 @@ const NavigationSideBar: React.FC = () => {
     const topNav = canAccessUncategorized
       ? [{ icon: 'experiment', label: 'Uncategorized', path: paths.uncategorized() }]
       : [];
+    const dashboardTopNav = [{ icon: 'home', label: 'Home', path: paths.dashboard() }];
```

Comment:
idk if this is a good idea, but `jupyter launch button` is moved in figma

https://www.figma.com/file/CwjY2phZfdZmlw6ijD18Bd/Home-%2F-Dashboard?node-id=117%3A13636&t=bzH38Ao3rYvyH7Lr-0

<img width="689" alt="image" src="https://user-images.githubusercontent.com/109113616/207957362-f18d1777-d9ed-4d0b-a30e-2884c238d904.png">

Reply:
making this change for when dashboard flag is on. also adding to Tasks page per discussion with @alexjdetermined